Cargo.toml

```diff
@@ -767,6 +767,7 @@ url = "2.5.4"
 usdt = "0.5.0"
 uuid = { version = "1.17.0", features = ["serde", "v4"] }
 uzers = "0.12"
+vergen = { version = "8.3.2", features = ["git", "git2"] }
```

Comment:
I see this is not the current release and the current release doesn't have some of these same interfaces.  Are we planning to stick with this version forever?

Reply:
I picked this version because it's the one we already have in `Cargo.lock` because it's used by `crucible-common` (https://github.com/oxidecomputer/crucible/blob/385be2e991708ead4fe9f28d57c66b841f2a92cd/common/Cargo.toml#L36). I think when I tried bumping it I ran into a cargo unsatisfiable dependency error; I'll try again. Maybe I should go bump it in crucible first?

Reply:
Turns out they didn't rip it out, exactly, just moved it to a separate crate. I opened https://github.com/oxidecomputer/crucible/pull/1770 to update Crucible, and can work around that for now by depending on their flavor that uses the git CLI instead of libgit. Done in c68985c56